www/src/index.ts

```diff
@@ -117,7 +120,8 @@ function main() {
         setTimeout(() => {
             map.beginAnimation();
         }, 0.5);
-    });
+    };
+    map.addEventListener(MapViewEventNames.FrameComplete, onFrameComplete);
```

Comment:
the fix look strange. Why are we using FrameComplete here? if it is for driving the animation we should use `AfterRender` or some other event